zproject/computed_settings.py

```diff
@@ -171,14 +171,14 @@ def get_dirs(self) -> List[Union[bytes, str]]:
     "zerver.middleware.TagRequests",
     "zerver.middleware.SetRemoteAddrFromRealIpHeader",
     "zerver.middleware.RequestContext",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
```

Comment:
As hinted by the comment a few lines up in this file, moving these before `LogRequests` will mean that these two middleware modules will not be counted in our analysis of how much time the request took to process. 

I think that might be OK -- the main thing this does that could consume resources is parsing cookies, which is unlikely to be significant to the runtime of the request.
```
    def process_request(self, request):                                                                  
        session_key = request.COOKIES.get(settings.SESSION_COOKIE_NAME)                                  
        request.session = self.SessionStore(session_key)                                                 
```